tests/e2e/subbyte_types/subbyte_types.mlir

```diff
@@ -0,0 +1,28 @@
+func.func @i1_type() {
+  %c0 = arith.constant 0 : index
+  %c255 = arith.constant 255 : i8
+  %input1 = util.unfoldable_constant dense<[85]> : tensor<1xi8> // 0,1,0,1,0,1,0,1
+  %input2 = util.unfoldable_constant dense<[170]> : tensor<1xi8> // 1,0,1,0,1,0,1,0
```

Comment:
I think using `0bxxxx` is better because it explicitly says that it's in binary form.

```suggestion
  %input1 = util.unfoldable_constant dense<[85]> : tensor<1xi8> // 0b010101...
  %input2 = util.unfoldable_constant dense<[170]> : tensor<1xi8> // 0b101010...
```

Reply:
What happens to this? It's not changed and it's marked resolved.